frame/benchmarking/src/lib.rs

```diff
@@ -242,7 +243,9 @@ macro_rules! benchmarks_iter {
 			{ $( $common )* }
 			( $( $names )* )
 			$name { $( $code )* }: {
-				<Call<T> as $crate::Dispatchable>::dispatch(Call::<T>::$dispatch($($arg),*), $origin.into())?;
+				<
+					Call<T> as $crate::frame_support::traits::PalletDispatchable
+				>::dispatch_bypass_filter(Call::<T>::$dispatch($($arg),*), $origin.into())?;
```

Comment:
```suggestion
				>::dispatch_bypass_filter(Call::<T>::$call_name($($arg),*), $origin.into())?;
```

Just a suggestion, I find it confusing since `Call::<T>::$dispatch($($arg),*)` reads like we're calling into a function named  `dispatch`, while we're selecting enum variants here.

Reply:
for consistency then we should rename $dispatch by $call_name everywhere in this macro